packages/suite-native-atoms/src/Input/Input.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { ReactNode, useRef, useState } from 'react';
+import { TouchableWithoutFeedback, TextInput } from 'react-native';
+
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { Text } from '../Text';
+import { Box } from '../Box';
+
+type InputProps = {
+    value: string;
+    label: string;
+    onChange: (value: string) => void;
+    hasError?: boolean;
+    hasWarning?: boolean;
+    leftIcon?: ReactNode;
+};
+
+type InputWrapperStyleProps = {
+    hasWarning: boolean;
+    hasError: boolean;
+};
+const inputWrapperStyle = prepareNativeStyle<InputWrapperStyleProps>(
+    (utils, { hasError, hasWarning }) => ({
+        borderWidth: utils.borders.widths.small,
+        borderColor: utils.colors.gray300,
+        backgroundColor: utils.colors.gray300,
+        borderRadius: utils.borders.radii.small,
+        paddingVertical: utils.spacings.small,
+        paddingHorizontal: 14,
+        height: 58,
+        justifyContent: 'center',
+        extend: [
+            {
+                condition: hasWarning,
+                style: {
+                    borderColor: utils.colors.yellow,
+                    borderWidth: utils.borders.widths.large,
+                },
+            },
+            {
+                condition: hasError,
+                style: {
+                    borderColor: utils.colors.red,
+                    backgroundColor: utils.transparentize(0.95, utils.colors.red),
+                },
+            },
+        ],
+    }),
+);
+
+const inputStyle = prepareNativeStyle(utils => ({
+    ...utils.typography.body,
+    alignItems: 'center',
+    justifyContent: 'center',
+    color: utils.colors.gray700,
+    padding: 0,
+    maxHeight: 24,
```

Comment:
why max height? why not just height?

Reply:
[bac2f35](https://github.com/trezor/trezor-suite/pull/5583/commits/bac2f35708091d37f278d3267c42c93a0c26edf9)